<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtkLineStyleWidget</class>
 <widget class="QWidget" name="vtkLineStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorEditWidget" name="colorWidget">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="styleLabel">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="styleComboBox">
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dashed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dotted</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
